cw_bitcoin/lib/electrum_wallet_addresses.dart

```diff
@@ -40,37 +55,50 @@ abstract class ElectrumWalletAddressesBase extends WalletAddresses with Store {
 
   static String toLegacy(String address) => bitbox.Address.toLegacyAddress(address);
 
-  final ObservableList<BitcoinAddressRecord> addresses;
+  final ObservableList<BitcoinAddressRecord> _addresses;
+  // Matched by addressPageType
+  late ObservableList<BitcoinAddressRecord> addresses;
```

Comment:
Modify it and make it less confusing please, not just with a comment, rename them.
and please double check their usage since they are so close, you might have mistakenly used one instead of the other